### PR TITLE
Make the Leviathan and Falcon more distinct

### PR DESCRIPTION
**Pull request by Anarchist2**
## Summary
The Falcon and the Leviathan are very similar ships: both cost more than 9 million credits, have the highest health among civilian ships and have 240 weapon space. However, despite costing 1 million more, and having 10 more required crew, the Falcon tends to be inferior to the Leviathan, having less health and significantly less outfit space. This PR aims to both buff the Falcon and make the Falcon and Leviathan more distinct ships, with the Falcon being a balanced ship with the ability to specialise and the Leviathan being specifically combat-focused.

Changes:

- Increased the Leviathan from 480 mass and 7.6 drag to 580 mass and 8.1 drag, and decrease the Falcon from 510 to 490 mass. This makes the speed difference between the two vessels far more noticable and impactful.
- Increased the Falcon's bunks from 75 to 91, and cargo from 90 to 130. This makes the Falcon a better transport, as well as making it better at capturing and plundering.
- Increased the Falcon's outfit space from 540 to 560. This brings the Falcon from having the worst outfit space of all obtainable Heavy Warships to being on par with the Vanguard. Even when considering the use of outfit expansions, the Falcon still comes out 30 outfit space short compared to an outfit-expanded Leviathan.
- Increased the Falcon's weapon space from 240 to 250. This extra weapon space allows the Falcon to mount stronger weaponry than the Leviathan, but its lower outfit space means that it will have to choose between mobility, utility and offence (and it will still have less HP than the Leviathan).
- Changed the Marauder Falcon's crew requirement from being 9 less than the original to being 5 higher. Every other Marauder has more crew requirement than the original.

Changes to the original models are also carried over to the Marauder variants.

### CODE REVIEW

File: data/human/marauders.txt [hunk 1]
```diff
@@ -228,15 +228,15 @@ ship "Marauder Falcon"
 		"cost" 13900000
 		"shields" 14100
 		"hull" 4100
-		"required crew" 43
-		"bunks" 80
-		"mass" 560
+		"required crew" 57
+		"bunks" 96
+		"mass" 540
```
Comment (Zitchas) [inline data/human/marauders.txt:233]:
```suggestion
		"mass" 560
```

File: data/human/marauders.txt [hunk 2]
```diff
@@ -477,8 +477,8 @@ ship "Marauder Leviathan"
 		"hull" 5500
 		"required crew" 48
 		"bunks" 69
-		"mass" 640
-		"drag" 7.6
+		"mass" 660
```
Comment (Zitchas) [inline data/human/marauders.txt:480]:
```suggestion
		"mass" 680
```

File: data/human/ships.txt [hunk 3]
```diff
@@ -1431,14 +1431,14 @@ ship "Falcon"
 		"shields" 12800
 		"hull" 3700
 		"required crew" 52
-		"bunks" 75
-		"mass" 510
+		"bunks" 91
+		"mass" 490
```
Comment (Zitchas) [inline data/human/ships.txt:1435]:
```suggestion
		"mass" 510
```

File: data/human/ships.txt [hunk 4]
```diff
@@ -2245,8 +2245,8 @@ ship "Leviathan"
 		"hull" 5000
 		"required crew" 43
 		"bunks" 64
-		"mass" 480
-		"drag" 7.6
+		"mass" 580
```
Comment (Zitchas) [inline data/human/ships.txt:2248]:
```suggestion
		"mass" 600
```

**Review by Zitchas — CHANGES_REQUESTED** [review]
Overall the changes look good for differentiating the two ships, however I don't think the falcon needs any reduction in mass. Better to just give the Leviathan and variants 20 additional mass than reduce the falcon by 20.

**Comment by williaji** [comment]
I'm not really a maths guy so these comments are from a player perspective. First, I would ask that you not make the Leviathan any slower than it is. It already feels limited to me on engine space and I've tried many varieties and it must be well balanced because I'm not pleased with any of the trade offs. Second, there are two changes to the Falcon that I would like to see: 1) enough engine/outfit space to be able to fit Orca engines. It always seems weird to me that there are engine sets that no ship can use, as if the designers had no access to the parts catalog. 2) Consider adding 2 gun ports. The weapon load out on both ship is identical, 4 turrets, 4 guns. Adding two gun ports is a clear break with the Leviathan.  would let you field more energy weapons while retaining missiles. it would allow you to tweak your outfit space some as guns are more efficient. It would  let you build a fast gunboat that is far superior to a Vanguard in terms of flexibility and range.

**Comment by Anarchist2** [comment]
Even in its stock configuration, the Leviathan is plenty agile, able to outturn almost every medium warship. When customising it, it is able to match either the Falcon's speed or turning while only being slightly down in the other category.

As for increasing the Falcon's engine space/hardpoint count, I don't see either of those changes as necessary or helpful. The Falcon is already very fast; letting it use both Orca engines doesn't really help it (nor do I see the argument that a ship that can mount both Orca engines is inevitable). Giving the Falcon more guns would only serve to remove the Vanguard's niche; besides, it's not like having the exact same number of hardpoints makes two ships inconvertibly similar. The Firebird, Osprey, Corvette and Argosy all have two turrets and four guns, yet all four ships feel very distinct.

**Pull request merged**